src/main/java/org/beanpod/switchboard/service/UserService.java

```diff
@@ -0,0 +1,33 @@
+package org.beanpod.switchboard.service;
+
+import lombok.RequiredArgsConstructor;
+import org.beanpod.switchboard.dao.UserDaoImpl;
+import org.beanpod.switchboard.dto.UserDto;
+import org.beanpod.switchboard.repository.UserRepository;
+import org.openapitools.model.UserModel;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class UserService implements UserDetailsService {
+
+  private final UserDaoImpl userDao;
+  private final UserRepository userRepository;
+  private final BCryptPasswordEncoder bCryptPasswordEncoder;
```

Comment:
To reduce the code smell related to `Member name 'bCryptPasswordEncoder' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'`. I'd suggest refactoring `bCryptPasswordEncoder` --> `bcryptPasswordEncoder` or `passwordEncoder`. Any name such that the second character is a lowercase letter, or number would be compliant.

Reply:
fixed